src/components/Comments/CommentsCard.jsx

```diff
@@ -17,8 +18,19 @@ export default function CommentCard({ comment, commentVotes }) {
       alert("your vote could not be added at this time");
     });
   };
+
+  const handleDeleteComment = (event) => {
+    event.preventDefault();
+    setHidden("hidden");
+    if (typeof comment.comment_id === "Number") {
```

Comment:
This should be lower case, I only spotted after I had merged.

Reply:
Will have to change this on main then! This branch is already merged so we can't go back and add more code to it